lib/utils/utils.js

```diff
@@ -213,12 +214,22 @@ function getSourceModuleNameForIdentifier(context, node) {
   const importDeclaration = program.body
     .filter(isImportDeclaration)
     .find(importDeclaration =>
-      importDeclaration.specifiers.some(specifier => specifier.local.name === node.name)
+      importDeclaration.specifiers.some(
+        specifier => specifier.local.name === getSourceModuleName(node)
+      )
     );
 
   return importDeclaration ? importDeclaration.source.value : undefined;
 }
 
+function getSourceModuleName(node) {
+  if (isMemberExpression(node) && node.object) {
+    return getSourceModuleName(node.object);
+  }
```

Comment:
Can you change this to:
```
} else if (isIdentifier(node)) {
  return node.name;
} else {
  assert(false, 'Function should only be called on `MemberExpression` or `Identifier`');
}
```
This will ensure the function is called on the right type of nodes (and make it clear what kind it can be used for).

Reply:
No worries. Done!